tools/blob_dump.cc

```diff
@@ -42,7 +46,9 @@ int main(int argc, char** argv) {
         fprintf(stdout,
                 "Usage: blob_dump --file=filename "
                 "[--show_key[=none|raw|hex|detail]] "
-                "[--show_blob[=none|raw|hex|detail]]\n");
+                "[--show_blob[=none|raw|hex|detail]] "
+                "[--show_raw_blob[=none|raw|hex|detail]] "
```

Comment:
Whats the difference between show_blob and show_raw_blob?

Reply:
I understand that show_raw_blob is for showing uncompressed blob data, but it is somewhat confusing as show_blob and show_raw_blob can again have a `raw` option value.

Reply:
The sst_dump_tool also refer to uncompressed data as "raw key" and "raw value" so I did so. :https://gist.github.com/yiwu-arbug/cdf5c3610fa04bd47c948fba8b02adef

Let me rename them.